components/images.css

```diff
@@ -0,0 +1,20 @@
+.eg-img-container {
+    background-color: var(--white);
+    border: 2px solid var(--secondary-200);
+    border-radius: 5px;
+    display: inline-block;
+    padding: 8px;
+}
+
+.img-responsive {
+    border-radius: 5px;
```

Comment:
You can use the values in the multiples of 4 or 8. Not necessary, just a suggestion.

Reply:
Done✅